scripts/bump_doc_dependencies.py

```diff
@@ -0,0 +1,96 @@
+# (C) 2023 GoodData Corporation
+import argparse
+from pathlib import Path
+
+import tomlkit
+
+_CURRENT_DIR = Path.cwd()
+
+_TOML_FILE = _CURRENT_DIR / "docs/config/production/params.toml"
+_REDIR_FILE = _CURRENT_DIR / "docs/layouts/index.redir"
+
+
+def bump_toml(file_path: Path, new_version: str):
+    if new_version.split(".")[-1] != "0":
+        print("params.toml not updated for hotfixes.")
+        return
+
+    new_version = ".".join(new_version.split(".")[:2])
+
+    # Load the TOML file
+    with open(file_path, "r") as file:
+        content = file.read()
+    data = tomlkit.parse(content)
+
+    versions = data.get("versions", [])
+
+    # Remove the old version
+    if versions:
+        versions.pop()
+
+    # Add the new version
+    new_version_data = tomlkit.table()
+    new_version_data["version"] = new_version
+    new_version_data["dirpath"] = new_version
+    new_version_data["url"] = f"/{new_version}/"
+    versions.insert(1, new_version_data)
+
+    # Write the updated data back to the TOML file
+    try:
+        with open(file_path, "w") as file:
+            file.write(tomlkit.dumps(data))
+    except Exception as e:
+        print("Error updating params.toml:", e)
+
+
+def bump_redir(file_path, new_version):
+    short_version = ".".join(new_version.split(".")[:2])
+
+    try:
+        new_redirect_rule = f"/{new_version}/ {{{{ .Site.BaseURL }}}}/{short_version} 301!\n"
+        with open(file_path, "r") as file:
+            content = file.readlines()
```

Comment:
Why do you read and write redir multiple times in the function? Could you rewrite it so it is used only once?